package.json

```diff
@@ -16,7 +16,7 @@
     "bluebird": "^3.4.7",
     "cheerio": "^0.22.0",
     "highlight.js": "^9.10.0",
-    "htmlparser2": "^3.9.2",
+    "htmlparser2": "https://github.com/MarkBind/htmlparser2#v3.10.0-markbind.1",
```

Comment:
Can we get rid of the `https://github.com/` part?

The "author/repo" format should be supported by [npm](https://docs.npmjs.com/files/package.json#github-urls)